src/acl/Manager.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 1996-2022 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#ifndef SQUID_ACLMANAGER_H
+#define SQUID_ACLMANAGER_H
+
+#include "acl/Acl.h"
+
+class ACLManager : public ACL
+{
+    MEMPROXY_CLASS(ACLManager);
+
+public:
+    ACLManager(char const *aClass) : class_(aClass) { context("manager", nullptr); }
```

Comment:
I have not checked how this context information is used, but perhaps we can supply "built-in" as the configuration source?

```suggestion
    ACLManager(char const *aClass) : class_(aClass) { context("manager", "built-in"); }
```